src/dmd/frontend.h

```diff
@@ -65,7 +65,6 @@ class StorageClassDeclaration;
 class ExpressionDsymbol;
 class AliasAssign;
 class ThisDeclaration;
-class BitFieldDeclaration;
```

Comment:
This is the result of running: `/tmp/.rdmd-1000/rdmd-build.d-F09C7D9CA23C97EB643AD504B23DCB35/build cxx-headers-test AUTO_UPDATE=1`

Reply:
You have to clean and rebuild build.d, or maybe jut remove all headers in the generated folder to force a rebuild.